data/successors/successor side missions.txt

```diff
@@ -0,0 +1,505 @@
+			`	Khosa-Kella-Oa itself is cold, dark and rocky, for all purposes functionally identical to any of the other innumerable uninhabited airless worlds in the galaxy. Your passengers seem pleased with it, though; Nnesa remarks that the local star "is so dim that there's hardly any glare!" and Jaasora energetically agrees. They activate their cart of folded equipment within the <ship>'s cargo bay, which slowly inflates and unfolds into something halfway between a tent and a repulsor lift. It skids over the floor like melting ice, gently bumping the inside walls a couple times before Jaasora manages to quiet its movement.`
+			`	"Our thanks again, captain, for transporting us," says Jaasora as they load a bundle of telescopes inside.`
+			`	"Oh! The planet's perfect; no air to muddy the view nor other ships to cloud the skies---"`
+			`	"We should be back in a couple hours. Please wait for us."`
+			`	The pair depart in their gliding tent, which manifests several transparent spots on its surface as it exits and as a bevy of telescopes are prodded around inside.`
+			choice
+				`	(Wait.)`
+			`	Nnesa and Jaasora return after a couple of hours as promised, their strange tent gliding back into the <ship>'s cargo bay as smoothly as it exited. The pair unload several notebooks of photos and drawings and chatter excitedly about the various constellations and stellar phenomena that they observed. Nnesa is particularly excited, hurriedly extracting from the interleaved mess of papers a surprisingly accurate and well-constructed star map, which they press up in your face with pride. You cannot help but notice that several of the drawings in the stack are of Jaasora rather than of any stellar phenomena.`
+			`	"From where on the map dost thou hail, captain?" Nnesa roughly gestures to an area of their map that you recognize as human space.`
+			choice
+				`	(Indicate Rutilicus.)`
+					goto rutilicus
+				`	(Indicate Sol.)`
+					goto sol
+				`	(Gesture broadly to the expanse of space depicted on the map.)`
+			`	"Oh!" says Nnesa, "Thou art a scion of the constellations, first and foremost to the stars? 'Tis fair. Thou hast a galaxy to wander, a nearly endless sky. The things that there must be to see---"`
+				goto interrupt
+			label rutilicus
+			`	"Oh!" says Nnesa, "I am not sure I know the name for that one, captain. What dost thou call thy home?"`
+			choice
+				`	"Rutilicus."`
+					goto rutilicus2
+			label rutilicus2
+			`	Nnesa asks you to repeat the word a second time, slowly working their way through your pronunciation and transcribing a short sequence of flowing letters on their map. "'Tis a pleasant name," they say.`
+				goto interrupt
+			label sol
+			`	"Oh!" says Nnesa, "Sol." They quietly sound their way through the word a couple times, trying to get it right. "Our name for it is Saaqa-Viirja. 'Twas one of the northernmost extents of our Predecessors' voyages. Ah. But thou know'st this already, for sure."`
```

Comment:
The Hai live far to the north from Sol, even if we count the wormhole. How come they are familiar with them?